mscore/editstyle.cpp

```diff
@@ -698,9 +698,31 @@ EditStyle::EditStyle(Score* s, QWidget* parent)
       if (isTooBig)
             this->setMinimumSize(scr.width() / 2, scr.height() / 2);
       hasShown = false;
+
+      adjustPagesStackSize(0);
+
       MuseScore::restoreGeometry(this);
       }
 
+void EditStyle::adjustPagesStackSize(int currentPageIndex)
+{
+    QSize preferedSize = pageStack->widget(currentPageIndex)->sizeHint();
```

Comment:
Should be `preferred`.

Reply:
and still the old code style, 6 space indent and banner style braces ;-)

Reply:
>Should be preferredSize.
Fixed

>and still the old code style, 6 space indent and banner style braces ;-)
It is a new code style. It will be easier for me to port this fix to master

Reply:
Yes, hence my ;-)